docs/en/rst/installing/mariadb.rst

```diff
@@ -0,0 +1,124 @@
+.. _mariadb:
+
+MariaDB
+#######
+
+It is strongly advised to use MariaDB which is fully compatible with Bugzilla.
+
+The minimum required version is MariaDB 10.0.5.
+
+It's possible to test which version of MariaDB you have installed with:
+
+:command:`mariadb -e 'select version()'`
+
+Installing
+==========
+
+Windows
+-------
+
+Download the MariaDB 32-bit or 64-bit MSI installer from the
+`MariaDB website <https://mariadb.org/download/?t=mariadb&os=windows>`_ (~66 MB).
+
+MariaDB has a standard Windows installer. It's ok to select a the
+default install options. The rest of this documentation assumes assume you
+have installed MariaDB into :file:`C:\\mysql`. Adjust paths appropriately if not.
+
+Linux/Mac OS X
+--------------
+
+The package install instructions given previously should have installed MariaDB
+on your machine, if it didn't come with it already. 
+
+If you did install MariaDB manually rather than from a package, make sure the
+server is started when the machine boots.
+
+Create the Database
+===================
+
+You need to create a database for Bugzilla to use. Run the :file:`mariadb` command-line client and enter:
+
+::
+    CREATE DATABASE IF NOT EXISTS bugs CHARACTER SET = 'utf8';
+
+The above command makes sure a database like that doesn't exist already.
+
+.. _mysql-add-user:
+
+Add a User
+==========
+
+You need to add a new MariaDB user for Bugzilla to use. Run the :file:`mariadb`
+command-line client and enter:
+
+::
+
+    GRANT SELECT, INSERT,
+    UPDATE, DELETE, INDEX, ALTER, CREATE, LOCK TABLES,
+    CREATE TEMPORARY TABLES, DROP, REFERENCES ON bugs.*
+    TO bugs@localhost IDENTIFIED BY '$DB_PASS';
+
+You need to replace ``$DB_PASS`` with a strong password you have chosen.
+Write that password down somewhere.
+
+The above command permits an account called ``bugs``
+to connect from the local machine, ``localhost``. Modify the command to
+reflect your setup if you will be connecting from another
+machine or as a different user.
+
+Change Configuration
+====================
+
+To change MariaDB's configuration, you need to edit your MariaDB
+configuration file, which is:
+
+* Red Hat/Fedora: :file:`/etc/my.cnf`
+* Debian/Ubuntu: :file:`/etc/mysql/my.cnf`
+* Windows: :file:`C:\\mysql\\bin\\my.ini`
+* Mac OS X: :file:`/etc/my/cnf`
```

Comment:
The config file on Mac is `/etc/my.cnf`. Is this wrong in the MySQL file as well? (I'm guessing this was just copied from there)

Reply:
According to MariaDB's docs, the config file can also be `mariadb.cnf` on Linux and Mac, but it'll use `my.cnf` if it's there. It still uses `my.ini` on Windows. https://mariadb.com/kb/en/configuring-mariadb-with-option-files/